src/metadata/metadata_superblock.c

```diff
@@ -303,12 +361,13 @@ struct ocf_pipeline_properties ocf_metadata_load_sb_recov_pipeline_props = {
 	.steps = {
 		OCF_PL_STEP_FOREACH(ocf_metadata_store_segment,
 				ocf_metadata_load_sb_store_segment_args),
+		OCF_PL_STEP_ARG_INT(ocf_metadata_load_segment,
+				metadata_segment_sb_config),
+		OCF_PL_STEP(_ocf_metadata_validate_superblock),
```

Comment:
this change belongs to previous commit, except that "_ocf_metadata_validate_superblock" would be ocf_metadata_check_crc_sb_config there.